src/oop_ext/interface/_interface.py

```diff
@@ -37,10 +37,32 @@ class MyCalculatorImpl(object):
 
```

Comment:
Could you please add `# mypy: disallow-untyped-defs` on this file?
There are some places without type annotation like line 77, 121, 130, 142

Reply:
Done!